Lesson2/SecondHomework.playground/Contents.swift

```diff
@@ -0,0 +1,50 @@
+/// An object, which is a list of elements with a type integer, organized according to the LIFO principle.
+class Stack {
+    fileprivate var array: [Int] = []
+    fileprivate var topElement: Int?
```

Comment:
First of all, can you please try to avoid using `topElement`?
You can perform push/pop/top operation using just `array` var.

Reply:
topElement was taken from an example on wikipedia. It can be removed and this does not affect the functions push/pop/top. In our case, an array would be enough